velox/connectors/hive/storage_adapters/s3fs/S3FileSystem.cpp

```diff
@@ -222,6 +222,31 @@ class S3Config {
         "hive.s3.iam-role-session-name", std::string("velox-session"));
   }
 
+  Aws::Utils::Logging::LogLevel getLogLevel() const {
+    auto level = config_->get("hive.s3.log-level", std::string("FATAL"));
```

Comment:
Should we move this to velox/connectors/hive/HiveConfig.h and document in https://facebookincubator.github.io/velox/configs.html#hive-connector ?

Reply:
The S3 FileSystem is an optional build. But the config class can live in the HiveConfig.h I presume.
Can we merge this PR as is? I will move all the `hive.s3.*` configs and document them in a follow-up PR.